polygon/rest/models/snapshot.py

```diff
@@ -31,6 +31,52 @@ def from_dict(d):
         )
 
 
+@modelclass
+class DayOptionContractSnapshot:
```

Comment:
the indices snapshot endpoint only returns open, high, low, close, change, change_percent, and previous_close. so we shouldn't need volume, VWAP, and last_updated

Reply:
im reusing the class, I just moved it up. its none, shouldn't return right?

Reply:
yeah it should be fine if you're reusing then, nvmd

Reply:
If this isn't being re-used can you move it back where it was to get it out of the diff

Reply:
@antdjohns This is the one thing.